ct_app/tests/original_test_SUITE.erl

```diff
@@ -1,6 +1,9 @@
 %% Original suite from the big_tests, which covers all cases.
 %% So, it is a bit complicated.
 %% Other suites test specific features and much simpler.
+%%
```

Comment:
I agree with that and never added these myself. That's also why I think it's worth to preserve git history - it's more truthful in case of files that are often changed.